uefi/src/data_types/owned_strs.rs

```diff
@@ -127,6 +201,12 @@ impl ops::Deref for CString16 {
     }
 }
 
+impl ops::DerefMut for CString16 {
```

Comment:
I'm not sure that CString16 should impl DerefMut, as that would make it possible to safely insert null characters in the middle of the string, which we've tried to prevent in other parts of the API.